demisto_sdk/commands/lint/lint_manager.py

```diff
@@ -565,16 +612,22 @@ def report_summary(self, lint_status: dict):
                                                  subsequent_indent=' ' * len(fail_pack_prefix))
         # intersection of all failed packages
         failed: Set[str] = set()
-        for packs in lint_status.values():
-            failed = failed.union(packs)
+        warnings: Set[str] = set()
+        for key in lint_status:
+            if key.startswith('fail'):
+                failed = failed.union(lint_status[key])
+            if key.startswith('warning'):
+                warnings = warnings.union(lint_status[key])
         # Log unit-tests summary
         sentence = " Summary "
         print(f"\n{Colors.Fg.cyan}{'#' * len(sentence)}")
         print(f"{sentence}")
         print(f"{'#' * len(sentence)}{Colors.reset}")
-        print(f"Packages: {len(self._pkgs)}")
-        print(f"Packages PASS: {Colors.Fg.green}{len(self._pkgs) - len(failed)}{Colors.reset}")
+        print(f"Packages: {len(pkg)}")
+        print(f"Packages PASS: {Colors.Fg.green}{len(pkg) - len(failed)}{Colors.reset}")
         print(f"Packages FAIL: {Colors.Fg.red}{len(failed)}{Colors.reset}")
+        print(f"Packages WARNING: {Colors.Fg.yellow}{len(warnings)}{Colors.reset}")
```

Comment:
Need to make sure this is true. as pkg can pass with warning, and pkg can fail and also have warnings
```suggestion
        print(f"Packages PASS: {Colors.Fg.green}{len(pkg) - len(failed) - len(warnings)}{Colors.reset}")
        print(f"Packages PASS WITH WARNINGS: {Colors.Fg.yellow}{len(warnings)}{Colors.reset}")
        print(f"Packages FAIL: {Colors.Fg.red}{len(failed)}{Colors.reset}")

```

Reply:
I was going for the option that no matter what if there are warnings than they will be printed and will be added to the summary report. However they do not fail lint in any way. 
Also , Fail and warnings can be shown together. For example, xsoar linter runs and there are failures as print exists in the code, but I would still want to show the warnings which are that there are still `demisto.results` usage and no main.

Reply:
So let's just add here
```
print(f"Packages WARNING(can either PASS or FAIL): {Colors.Fg.yellow}{len(warnings)}{Colors.reset}")
```